modules/base/shaders/renderabletrail_fs.glsl

```diff
@@ -22,19 +22,28 @@
  * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
  ****************************************************************************************/
 
-uniform float forceFade;
-uniform vec3 color;
+// Fragile! Keep in sync with RenderableTrail::render::RenderPhase 
+#define RenderPhaseLines 0
+#define RenderPhasePoints 1
 
 in vec4 vs_positionScreenSpace;
 in float fade;
 
-#include "PowerScaling/powerScaling_fs.hglsl"
+uniform vec3 color;
+uniform int renderPhase;
+
 #include "fragment.glsl"
 
 Fragment getFragment() {
-    vec4 c = vec4(color * fade * forceFade, 1.0);
+    if (renderPhase == RenderPhasePoints) {
+        vec2 circCoord = 2.0 * gl_PointCoord - 1.0;
+        if (dot(circCoord, circCoord) > 1.0) {
```

Comment:
Nice way to check if length > 1. I assume it's faster since there is no square root involved. Maybe deserves a commend to explain? I have not checked the visual result yet, but it looks like this could cause jagged edges? How about something like: 
`alpha *=  smoothstep(dot(circCoord, circCoord),  1, 1 - 1.0 / pointSize) `